packages/fiori/test/specs/ViewSettingsDialog.spec.js

```diff
@@ -38,13 +40,14 @@ describe("ViewSettingsDialog general interaction", () => {
 	it("test ViewSettingsDialog - sortBy confirm selected settings", async () => {
 		const btnOpenDialog = await browser.$("#btnOpenDialog");
 		const viewSettingsDialog = await browser.$("#vsd");
+		const input = await browser.$("#sortByItem");
 		await btnOpenDialog.click();
 
 		assert.notOk(await viewSettingsDialog.shadow$("[sort-by]").$("ui5-li[selected]").isExisting(), "sortBy should not have an option selected");
 
 		await viewSettingsDialog.shadow$("[sort-by]").$("ui5-li").click();
 		await viewSettingsDialog.shadow$("ui5-dialog").$(".ui5-vsd-footer").$("ui5-button").click();
-
+		assert.equal(await input.getAttribute("value"), "UI5-SORT-ITEM", "sortByItem should return HTML element");
```

Comment:
We need to verify that it returns the selected HTML element, not just a HTML element, otherwise the test is not that useful.

Reply:
done